packages/xml-views-quick-fix/src/quick-fix-id.ts

```diff
@@ -0,0 +1,107 @@
+import { find, some } from "lodash";
+import { astPositionAtOffset } from "@xml-tools/ast-position";
+import {
+  XMLDocument,
+  accept,
+  XMLAstVisitor,
+  XMLAttribute,
+  XMLElement,
+} from "@xml-tools/ast";
+import { OffsetRange } from "@ui5-language-assistant/logic-utils";
+
+const ID_PATTERN = /^_IDGen(.+)([1-9]\d*)$/;
+const ID_PREFIX_PATTERN = "_IDGen";
+
+export type QuickFixStableIdInfo = {
+  suggestion: string;
+  offsetRange: OffsetRange;
+};
+
+export function computeQuickFixStableIdInfo(
+  xmlDoc: XMLDocument,
+  errorOffset: OffsetRange
+): QuickFixStableIdInfo | undefined {
+  const biggestIdsCollector = new IdsByElementNameCollectorVisitor();
```

Comment:
the line:
`const biggestIdsCollector = new IdsByElementNameCollectorVisitor();`

Hints the class name of the visitor is inaccurate,
Perhaps it should be more similar to the variable name on the left hand side